src/audio/audio.c

```diff
@@ -1714,7 +1714,8 @@ void audio_init1()
 			}
 		}
 		m++;
-	} while(m + 1 < 0x2E);
+	// 46 is the number of files in ride_music_info_list
+	} while(m + 1 < 46);
```

Comment:
countof is the macro. The music list is defined in ride\ride.c. I think that macro will work with it but have a test of it since unsure how it works with externs.